### ISSUE TITLE
In file transfers : Agrupate uploads of the same file in one line like downloads

### ISSUE THREAD

**Issue by felisucoibi** [RetroShare/RetroShare#1012]
In file transfers : Agrupate uploads of the same file in one line like downloads, that means one line for each file, ATM there are usually like 5 lines of different anonymous tunnels for the same file.... that makes 20 files 2x5 lines, lot of lines.... ugly, very ugly.... user worries about that and dont like retroshare.... i asked for limited tunnels, ok, no limited tunnels, at least agrupate them.

**Comment by felisucoibi** [comment]
I update, its done but needs more things....

I update the post.

ITS DONE:
In file transfers : Agrupate uploads of the same file in one line like downloads, that means one line for each file, ATM there are usually like 5 lines of different anonymous tunnels for the same file.... that makes 20 files 2x5 lines, lot of lines.... ugly, very ugly.... user worries about that and dont like retroshare.... i asked for limited tunnels, ok, no limited tunnels, at least agrupate them.

BUT THERE IS ONE THING MISSING:
If there is only one user one connection downloading a file, dont agrupate, so with just one sight you can see the progress bar of upload, so with one sight you can see how more is missing to be uploaded.
If there are some users, usually anon users, agrupate.
This is important because usually direct connections use only one conection / file and is very usefull to see how much needed, and with anons is the oposite.

I can even say, if is direct connect and is 2 connections for the same user, agrupate, but show the progress column outside the tree in the , not necessary to desplegate to see progress.


In short agrupate when needed, some direct connections to the same file, or some anonimous connections, but if it's only one user, show progress, not agrupate.

**Comment by felisucoibi** [comment]
Done
